analyzers/src/SonarAnalyzer.CSharp/Rules/SymbolicExecution/SymbolicExecutionRunner.cs

```diff
@@ -56,6 +57,14 @@ public class SymbolicExecutionRunner : SymbolicExecutionRunnerBase
             new InitializationVectorShouldBeRandom(),
             new HashesShouldHaveUnpredictableSalt());
 
+        public SymbolicExecutionRunner() : this(AnalyzerConfiguration.AlwaysEnabledWithSonarCfg)
+        {
+        }
```

Comment:
All of these can be on a single line
```suggestion
        public SymbolicExecutionRunner() : this(AnalyzerConfiguration.AlwaysEnabledWithSonarCfg) { }
```